src/core/AutoRest.Core/Properties/Resources.resx

```diff
@@ -252,4 +252,13 @@
   <data name="InvalidConstraint" xml:space="preserve">
     <value>Constraint is not supported for this type and will be ignored.</value>
   </data>
+  <data name="NonAppJsonTypeNotSupported" xml:space="preserve">
+    <value>Non application json MIME type has limited support</value>
```

Comment:
How about `Support for media types other than "application/json" has limited support` 

Points to note: 
- be explicit about what someone should use (`"application/json"` )
- `MIME type` is actually not the correct term anymore . It's `Media Type` (it not being used to describe Multipurpose Internet Mail Extensions anymore...)